src/jit/emit.cpp

```diff
@@ -7277,3 +7303,41 @@ const char* emitter::emitOffsetToLabel(unsigned offs)
 }
 
 #endif // DEBUG
+
+//------------------------------------------------------------------------
+// GetSavedSet: Returns the set of registers that live across the call.
+//
+// Arguments:
+//   methHnd - the method handler of the call.
+//
+// Return value:
+//   the saved set of registers.
+//
+regMaskTP emitter::GetSavedSet(CORINFO_METHOD_HANDLE methHnd)
```

Comment:
IMO, we should maintain the existing pervasive emitter naming style and name this `emitGetSavedGCRegsSet`

Reply:
Fixed